styles.css

```diff
@@ -41,15 +42,15 @@ aside{
 article.subArticleC{
     box-sizing: border-box;
     grid-area: C;
-    background-color: darkorchid;
+    background-color: #ac4449;
     display: flex;
     justify-content: center;
     align-items: center;
 }
 article.subArticleD{
```

Comment:
camelCase niet in CSS alleen in Java. Zeer verwarrend al zeg ik het zelf, maar staat beschreven in het nakijkblaadje van Nova.